3-JSS/main.js

```diff
@@ -0,0 +1,182 @@
+import { v4 as uuidv4 } from "uuid";
```

Comment:
has this file changed besides the way classes are imported? cause i'm not reviewing it 😅

Reply:
lol, I just change how the classes are imported 😅